src/components/ScenariosButtons.tsx

```diff
@@ -1,16 +1,13 @@
 import React from 'react';
-import { Navbar, Nav, Form, FormControl, Button, Container, ButtonGroup, Card } from 'react-bootstrap';
+import { Button, Container, Card } from 'react-bootstrap';
 import { HttpClient } from '@butter-robotics/mas-javascript-api';
-import { start } from 'repl';
 
 const kip1 = new HttpClient('192.168.57.30');
 const kip2 = new HttpClient('192.168.56.188');
 
 kip1.timeout = 240;
 kip2.timeout = 240;
 
-let interval1: NodeJS.Timeout;
-let interval2: NodeJS.Timeout;
```

Comment:
aren't we using those intervals? did you perform a live test on few robots to verify all functioning well?

Reply:
The PR deploy test failed when I had warnings in the code, such as "defined but never used", so i have to delete this never used line. 
I didn't check it in the lab, but we can keep this lines, as I said, only removed them to get rid of the warnings to pass the deploy tests.